### DISCUSSION TITLE
Bayesian GPLVM -- lower bound

### DISCUSSION THREAD

**Discussion by vr308** [GPflow/GPflow#1705] [General]
Hi  - I had a question about the Bayesian GPLVM elbo, my guess was that the elbo is as per the original Titsias paper 
http://proceedings.mlr.press/v9/titsias10a/titsias10a.pdf , equation 14 but I think whats implemented is a slightly different version?

Is this to enable mini-batching?

**Comment by st--** [comment]
The GPLVM was originally implemented by @markvdw and @alexisboukouvalas (see https://github.com/GPflow/GPflow/pull/91 and https://github.com/GPflow/GPflow/pull/195) - either of them would have to comment on this: )
